src/main/java/com/google/sps/seeder/AppSeeder.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.seeder;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+public class AppSeeder {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    public void seedDB() {
+        setEntityProperties("social", "connect with friends", "mockIcon.url", "com.facebook.ocra", "Messenger", 0, 4.6);
+        setEntityProperties("music", "Listen to your songs", "mockIcon.url", "com.pandora.android", "Pandora", 0, 4.6);
+        setEntityProperties("wearOS", "Be in control of you WearOS", "mockIcon.url", "com.google.android.wearable.app", "Wear OS by Google", 0, 4.6);
```

Comment:
Lets make at least one of these mock apps 0.99 for price. there are some frontend UI that only shows up when the price is greater than 0